test/lib/processors/libraryLessGenerator.js

```diff
@@ -0,0 +1,483 @@
+const test = require("ava");
+const sinon = require("sinon");
+
+const processor = require("../../../lib/processors/libraryLessGenerator");
+const {_LibraryLessGenerator: LibraryLessGenerator} = processor;
+
+const FILE_HEADER = `/* NOTE: This file was generated as an optimized version of ` +
+	`"library.source.less" for the Theme Designer. */`;
+
+test.afterEach(() => {
+	sinon.restore();
+});
+
+test.serial("processor", async (t) => {
```

Comment:
Why are all the tests serial? I don't see how they could interfere with one another.

Reply:
I think I've added it because of the `sinon.restore()` in the `afterEach` hook.
But restoring isn't required as no existing functions are spied/stubbed.